client/components/header-cake/back.jsx

```diff
@@ -63,13 +64,10 @@ export default React.createClass( {
 		} );
 
 		return (
-			<a className={ linkClasses } href={ href } onClick={ onClick }>
-				<Gridicon icon="chevron-left" size={ 18 } />
-				{
```

Comment:
I'm a little fuzzy on what this logic did, I was getting an error when I tried including it, so I removed it. Could someone (maybe @mtias double-check that I haven't broken something by removing it?)